### DISCUSSION TITLE
Support for multiple indices with plugin to route data.

### DISCUSSION THREAD

**Discussion by ravin-accelq** [toluaina/pgsync#420] [Ideas]
So the Idea here is to have index property in schema.json file, to accept an array of indices and then using the plugin extensibility segregate data into different buckets where each bucket of data is sent into an index.

So schema would look like {
 "indices": ["index_1", "index_2"]
}

and the plugin returns a tuple (doc, index_id)

**Comment by toluaina** [comment]
You could also just have multiple nodes in your schema.json

e.g
```
[
    {
        "database": "book",
        "index": "book",
        "nodes": {
            "table": "book",
            "columns": [
                "id"
            ]
        }
    },
    {
        "database": "book",
        "index": "book2",
        "nodes": {
            "table": "book",
            "columns": [
                "id"
            ]
        }
    }
]
```
